Runtime/EOSSDK/SDK/Plugins/EOSSDK-Win32-Shipping.dll.meta

```diff
@@ -16,22 +16,32 @@ PluginImporter:
     second:
       enabled: 0
       settings:
+        Exclude Android: 1
         Exclude Editor: 0
         Exclude Linux64: 0
         Exclude OSXUniversal: 0
+        Exclude WebGL: 1
         Exclude Win: 0
         Exclude Win64: 1
+        Exclude iOS: 1
+  - first:
+      Android: Android
+    second:
+      enabled: 0
+      settings:
+        AndroidSharedLibraryType: Executable
+        CPU: ARMv7
   - first:
       Any: 
     second:
-      enabled: 1
+      enabled: 0
       settings: {}
   - first:
       Editor: Editor
     second:
       enabled: 1
       settings:
-        CPU: x86
+        CPU: AnyCPU
```

Comment:
Shouldn't this remain as x86 since it's a Win32 DLL?

Reply:
Yeah, it's the same like with xaudio dll below, Unity changed it automatically, should be good since there is no x86 support in the editor.